open-craft/problem-builder
# Update requirements to support Django 2.2

This updates problem-builder to support Django 2.2.

**JIRA tickets**: Implements BB-2061.

**Testing instructions**:

1. Follow XBlock workbench installation instructions
2. Check example problems in workbench.
3. Run integration and quality tests.

**Author notes and concerns**:

1. There are some exceptions when running integration tests, these also exist in the Django 1.11 version, but are all related to undefined variables in the template's context which are only shown in DEBUG mode or in tests.

**Reviewers**
